modules/shared/src/main/scala/org/tessellation/tools/DAGBlockGenerator.scala

```diff
@@ -0,0 +1,67 @@
+package org.tessellation.tools
```

Comment:
imo it should be in `test-shared` and named differently than `tools`, `utils`, etc

Reply:
I followed the placement of TransactionGenerator. Should it be changed too?

Reply:
good question, probably to author of TxGenerator. can we discuss that internally on slack with the whole team?

Reply:
bump @TheMMaciek @tbekas @kpudlik